Add README (#6)
ManageIQ/manageiq-rpm_build: README.md

# ManageIQ RPM Build

> This repository contains code to build RPMs for ManageIQ appliances in a container image:

Reviewer: ```suggestion
This repository contains code to build RPMs for ManageIQ appliances in a appliances and images:
```

> This repository contains code to build RPMs for ManageIQ appliances in a container image:

Reviewer: ```suggestion
This repository contains code to build RPMs for ManageIQ appliances and container images:
```
